embedder.py

```diff
@@ -0,0 +1,152 @@
+from gensim.models import Word2Vec
+import pandas as pd
+import multiprocessing
+import itertools
+import os
+from typing import Iterator, List, Callable
+
+from time import time
+import logging  # Setting up the loggings to monitor gensim
+logging.basicConfig(format="%(levelname)s - %(asctime)s: %(message)s", datefmt= '%H:%M:%S', level=logging.INFO)
```

Comment:
For some reason it logs everything on the error output stream (or at least in red), not a huge issue of course.

Reply:
I removed this specific logging now since it was only there to test if Word2Vec was actually working.